core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/ConfigParameters.scala

```diff
@@ -117,6 +117,14 @@ object IntegerConfigParameter {
 final case class IntegerConfigParameter(key: String, description: String = "", defaultValue: Option[Int] = None) extends ConfigParameter {
   def toDescriptor: ConfigParameterDescriptor = ConfigParameterDescriptor(key, description, IntegerValidationType, defaultValue)
 
+  /**
+   * Java API
+   * Gets the value for this configuration parameter.
+   */
+  def getValue(context: StreamletContext): Int =
+    context.streamletConfig.getInt(key)
+  def value(implicit context: StreamletContext) =
```

Comment:
See examples, you can now just do (where Throttle is a ConfigParameter) in StreamletLogic:
```
val throttleValue = Throttle.value
```